indigo-frontend/src/app/pages/project/project-info/project-info.component.ts

```diff
@@ -12,9 +12,15 @@ import { of, Subject, take } from 'rxjs';
 import { takeUntil, catchError } from 'rxjs/operators';
 import { FileUploadComponent } from "@/core/components/common/file-upload/file-upload.component";
 import { Attachment } from '@/core/types/entities/attachment.i';
-import { MatDialog } from '@angular/material/dialog';
+import { MatDialog, MatDialogRef } from '@angular/material/dialog';
 import { ProjectAddComponent } from '../project-add/project-add.component';
+import { NotebookAddComponent } from '../notebook/notebook-add/notebook-add.component';
+import { ProjectOverviewWidgetDirective } from '../../../../core/components/project/projects-overview-widget/directives/project-overview-widget.directive';
 
+enum projectInfoModalEnum {
```

Comment:
By naming convention, enums should be started with capital letter. Also, we don't need postfix Enum I guess.